dashboard/app/models/pd/application/teacher_application.rb

```diff
@@ -398,7 +398,7 @@ def deliver_email(email)
       end
 
       # email_type maps to the mailer action
-      TeacherApplicationMailer.send(email.email_type, self)
+      TeacherApplicationMailer.send(email.email_type, self).deliver_now
```

Comment:
confusingly, "send" in this context is not a mailer command and has nothing to do with sending email, rather it is calling Object#send, which is ruby's way of saying "call the method named #{email_type}", such as one of these: 
* https://github.com/code-dot-org/code-dot-org/blob/fd70ad374f46aa1ef1012ac03549227e71076619/dashboard/app/mailers/pd/application/teacher_application_mailer.rb#L82
* https://github.com/code-dot-org/code-dot-org/blob/fd70ad374f46aa1ef1012ac03549227e71076619/dashboard/app/mailers/pd/application/teacher_application_mailer.rb#L92

Reply:
Yes, that tripped us up! It wasn't until we couldn't find a `send` method in ActionMailer that we realized this has nothing to do with a mailer action. Thanks for clarifying!